service/src/main/java/bio/terra/catalog/common/DaoKeyHolder.java

```diff
@@ -0,0 +1,42 @@
+package bio.terra.catalog.common;
```

Comment:
Are these files generated? From the swagger stuff?  Just curious, not asking for change

Reply:
There are many files generated by swagger, but we don't check them in. The swagger task will regenerate the files as prerequisite to the java compile task if necessary.